### PR TITLE
update VZ dates

### PR DESCRIPTION
**Pull request by rosemarymarano**
Add End of Error Correction dates to Prereqs doc.

### CODE REVIEW

File: content/en/docs/setup/prereqs.md [hunk 1]
```diff
@@ -20,12 +20,12 @@ Verrazzano supports the following software versions.
...
 
-| Verrazzano | Release Date | Latest Patch Release | Latest Patch Release Date | Kubernetes Versions |
-|------------|--------------|----------------------|---------------------------|------------------|
-| 1.0        | 2021-08-02   | 1.0.4                | 2021-12-20                | 1.18, 1.19, 1.20 |
-| 1.1        | 2021-12-16   | 1.1.2                | 2022-03-09                | 1.19, 1.20, 1.21 |
-| 1.2        | 2022-03-14   | 1.2.2                | 2022-05-10                | 1.19, 1.20, 1.21 |
-| 1.3        | 2022-05-24   | 1.3.3                | 2022-07-28                | 1.21, 1.22, 1.23 |
+| Verrazzano | Release Date | Latest Patch Release | Latest Patch Release Date | Kubernetes Versions | End of Error Correction |
+|------------|--------------|----------------------|---------------------------|---------------------|-------------------------|
+| 1.0        | 2021-08-02   | 1.0.4                | 2021-12-20                | 1.18, 1.19, 1.20    | 2022-06-30
```
Comment (gilbode) [inline content/en/docs/setup/prereqs.md:25]: should we either remove 1.0 b/c it's obsolete or put it an a separate list of EOL releases?

Comment (gilbode) [reply content/en/docs/setup/prereqs.md:25]: Actually - perhaps we just reverse the order of the table (most recent version first).

Comment (rosemarymarano) [reply content/en/docs/setup/prereqs.md:25]: So, in other words, keep all the data the same, including all the End of Error Correction dates, and just reverse their order in the table? I can do that.

Comment (gilbode) [reply content/en/docs/setup/prereqs.md:25]: yes - that way the first thing you see is the current (supported release)

**Review by gilbode — APPROVED** [review]

**Pull request merged**